### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Khaled-Alaa-1**
In this PR, I did the following:

- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

- In order to lay out the elements on the page you should use Grid and Flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,10 +21,16 @@
...
+          </UL>
       </header>
       <div class="main">
         <h1>
-           I<span>&#39;</span>m Khaled Glad to see you! 
+           I<span>&#39;</span>m Khaled 
+           <br> Glad to see you! 
         </h1>
         <p class="Main-Text">
           I<span>&#39;</span>m a software developer! I can help you build a
```
Comment (Shedrack-Sunday) [inline index.html:36]: - [x] Good job in this section 👏 . However, it seems your background image has an overlap that clusters the background. the initial figma design has just one, which makes it simple yet professional and with a great user experience. Kindly adjust this based on the comparison below, to complement your great work so far 👍 

## Your view :
image

## Figma:
image

File: MIT.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
 ## Copyright 2021, [Khaled alaa]
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
-
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```
Comment (Shedrack-Sunday) [inline MIT.md:4]: - [x] You did well adding the Licenee file, but it is displaying an error when clicked. please check the file path and make sure it is viewable on click 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Khaled-Alaa-1     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required have to do with a bit of styling and the License file 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Khaled-Alaa-1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**